SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/CallbacksTest.cpp

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialView/Callbacks.h"
+
+#define CALLBACKS_TEST(TestName) GDK_TEST(Core, Callbacks, TestName)
+
+using namespace SpatialGDK;
```

Comment:
Nit - even in cpp files I'd prefer to not have this. Slightly better is using specific types rather than the whole namespace.

However for tests why not just put them in the same namespace?

Reply:
the namespace thing was copied for the worker view test, which I based this off- although I see why it is an issue. Are you saying it should instead all be defined inside the GDK namespace, or all types should be prefixed?

Reply:
changed to specify for each type as it felt nicer